tox.ini

```diff
@@ -14,7 +14,7 @@ envlist =
 [testenv]
 deps = {[base]testdeps}
 extras = plotly
-commands = pytest {posargs}
+commands = pytest {posargs} tests/
```

Comment:
Why? Adding this will prevent us to use a command like `tox -e py38 -- tests/test_something.py`.
I think it would be better to add this in the `pyproject.toml` file:
```toml
[tool.pytest.ini_options]
testpaths = [
    "tests",
]
```
So by default it only uses the tests from the `tests` directory but it is still possible to specify which tests to run with tox.

Reply:
Indeed. Thanks for the suggestion. I added it.